### PR TITLE
Smashing branch

### PR DESCRIPTION
**Pull request by Alaska01**
-Made thought on visual hierarchy, typography and design principles for this project.
- Created a grayscale heat map of the website that indicates which elements have the greatest weight in the visual hierarchy.
-Created a new HTML document.
-Made background colors of those divs different shades of gray which correspond to their visual weight on the Smashing homepage.
-Copy the basic structure of the page using empty semantic HTML elements that are appropriately sized and positioned.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: index.html [hunk 1]
```diff
@@ -1,13 +1,540 @@
...
+        </div>
+      </div>
+    </div>
+
+    <div class="latest-articles light-gray-bg">
+      <p><a href="#">SEE ALL THE LATEST ARTICLES →</a></p>
+    </div>
+
+    <div class="grid-four dark-gray-bg">
+      <div class="grid-4box-1">
```
Comment (andresporras3423) [inline index.html]: Replace div with article tag

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+
+.col-1a {
+  grid-column: 1/2;
+  grid-row: 1/3;
+}
+
+.col-2a {
+  grid-column: 2/-1;
+  grid-row: 1/-1;
+  background-color: #333; /* background-color: rgba(3, 3, 3, 0.3); */
```
Comment (andresporras3423) [inline css/styles.css]: Remove commented code in this line

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Alaska01 and @Godswilly** 
Good job with your first submission for this project :muscle: your page meets most of the requirements :clap: :clap:
There are just a few details to improve before you can continue:
- [ ] your README file doesn't have a valid description. Explain whats is this project about.
- [ ] remove commented code in your CSS file
- [ ] Check below, I point in your code where you should replace div with the article tag
- [ ]  For the navbar, use nav element with ul/li inside
- [ ] All the elements and element's backgrounds and font colors should be grayscale color (including images). You can use images in this project, but just grayscale.
- [ ] The footer must contain centered links, check out the original page  :point_down: :point_down: :point_down:: 





Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by Alaska01** [comment]
Worked on the readme file, removed CSS comments, replaced divs in the navbar with ul/li, centered the footer links, changed the div tag to article tag, and made use of grayscale color for all backgrounds elements and font colors as requested by the TSE.

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨
## Impressive project @Alaska01 and @Godswilly  ✨

Great job applying the changes. However, a small part in the `footer` still requires some change.

File: index.html [hunk 3]
```diff
@@ -1,13 +1,570 @@
...
+      <li><a href="#">Design</a></li>
+      <li><a href="#">Design Patterns</a></li>
+      <li><a href="#">Design Systems</a></li>
+      <li><a href="#">E-Commerce</a></li>
+      <li><a href="#">Freebies</a></li>
+      </ul>
+
+  </div>
+   
+  <div class="sub-footer">
```
Comment (tirthajyoti-ghosh) [inline index.html:543]: The footer must have two placeholders for the two images on both sides.
Original 👇
Annotation 2020-06-06 180247

Right now 👇
Annotation 2020-06-06 180209

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply this small change and request for another review.

Happy coding 👨‍💻

File: index.html (hunk 3 above)
Comment (Alaska01) [reply index.html:543]: Thanks for the review. Have updated the two images in the footer section. Please note that the smashing magazine images are changing from time to time as the site is opened. We have updated two images to demonstrate that the images can be positioned in that position.

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## Congratulations. Your project is approved!! 🙌👏

Charge onto your next project 🚀

Good luck!! 👍

**Pull request merged**